.travis.yml

```diff
@@ -52,7 +35,7 @@ jobs:
       # Disable pip cache dir temporarily
       install:
        - pip install --no-cache-dir -r requirements-selftests.txt
-    - python: "3.8"
+    - python: "3.9"
       arch: ppc64le
```

Comment:
So, ppc64le is one of the architectures that consume credits. The free archs are s390x and arm. Are we keeping ppc? (cc @beraldoleal @clebergnu @richtja)

Reply:
@willianrampazzo  During my build tests of this PR in my userspace, none of the three arches have consumed credits. According to the documentation, these are offered for free from partners:
https://docs.travis-ci.com/user/billing-overview/#partner-queue-solution

Reply:
Ah, thanks for checking that! When I first read this documentation, I thought IBM was supporting only s390x. So, we are good here!